scripts/ec2/deploy_tags.coffee

```diff
@@ -0,0 +1,58 @@
+# Description:
+#   Provides conveniences around certain tags we want to manage
+#
+# Commands:
+#   hubot ec2 reserve <instance_id> [<instance_id> ...] --ReservationUser --ReservationBranch --ReservationDescription
```

Comment:
we can remove `[<instance_id> ...]` since we only take on instance ID; we can also make it clearer that you need to `--ReservationUser=<username>` etc